src/redux/features/tracking/trackingSelectors.ts

```diff
@@ -1,8 +1,25 @@
 import { RootState } from 'src/redux/rootReducer';
 import { PlantingSite } from 'src/types/Tracking';
+import { createCachedSelector } from 're-reselect';
 
 export const selectPlantingSites = (state: RootState) => state.tracking?.plantingSites;
 export const selectPlantingSitesError = (state: RootState) => state.tracking?.error;
 
 export const selectPlantingSite = (state: RootState, plantingSiteId: number) =>
   selectPlantingSites(state)?.find((site: PlantingSite) => site.id === plantingSiteId);
+
+export const selectZoneProgress = createCachedSelector(
```

Comment:
@charlie-tf just curious.. can this be used in @constanzauanini 's PR later?
See the BE [PR](https://github.com/terraware/terraware-server/pull/1293) that was updated for the nursery withdrawals plantings redux WIP [PR](https://github.com/terraware/terraware-web/pull/1558).

Reply:
I think this is by zone, I need it by subzone

Reply:
Yeah, this selector aggregates by zone